### DISCUSSION TITLE
For some reason, my components are not statically rendered

### DISCUSSION THREAD

**Discussion by stefan-h-singer** [react-static/react-static#1445] [General] [answered]
I don't even know where to start looking, I _think_ this used to work in the past but as of right now, Only the header and footer of the site have content in the generated html, the rest is rendered dynamically. 

Any ideas as to where I should start looking?

**Comment by stefan-h-singer** [comment]
To clarify, `<Routes/>` does not render statically, but anything outside of it is fine.

**Comment by stefan-h-singer** [comment]
I've also been looking at the default project created when using react-static create and it has got the same issue, nothing within the Routes component is rendered statically.

**Comment by SleeplessByte** [comment]
It renders statically in a production build. It is on demand in development.

**Reply by stefan-h-singer** [reply]
I know. I've been testing both react-static build and react-static build --staging. The templates for the routes does not get prerendered.

**Comment by stefan-h-singer** [comment]
This seems to be an issue with 7.3.0, using 7.2.3 it seems to work. Will do some further testing.

**Reply by SleeplessByte** [reply]
Are you on Windows, Mac or Linux? Might be related.

cc @nddery

**Reply by stefan-h-singer** [reply]
OS X and docker instance under Ubuntu, same result.

**Reply by stefan-h-singer** [reply]
Verified issue on separate computer as well, so on three different systems.

**Comment by stefan-h-singer** [comment, marked as answer]
Yup. Downgrading to react-static@7.2.3 fixes the issue. That was a tricky one to track down.

**Reply by SleeplessByte** [reply]
Should be fixed in 7.4.0
